### PR TITLE
SC-5741 - cleanup

### PR DESCRIPTION
**Pull request by BlackGhostHD**
# Description

## Links to Tickets or other pull requests

- https://ticketsystem.schul-cloud.org/browse/SC-5741
- https://github.com/hpi-schul-cloud/schulcloud-client/pull/2025

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by BlackGhostHD** [comment]
@schul-cloud-bot update screenshots

File: src/components/legacy/LegacyFooter.vue [hunk 1]
```diff
@@ -31,26 +31,18 @@
...
 						Du möchtest gerne Mitglied in unserem HPI
 						<br />HPI Schul-Cloud Team werden? Wir suchen:
 					</p>
-					<base-link
+					<!-- <base-link
 						class="link-style"
 						href="https://hpi.de/das-hpi/organisation/jobs/aktuelle-jobs/internettechnologien/web-developer-mwd-im-projekt-hpi-schul-cloud.html"
 						>Web Developer</base-link
-					>
+					> -->
```
Comment (CordlessWool) [inline src/components/legacy/LegacyFooter.vue:45]: Why do you not remove it if it is not necassery anymore?

Comment (BlackGhostHD) [reply src/components/legacy/LegacyFooter.vue:45]: It was commented out on also the legacy client. I don't know the reason for this,so I did it the same way here

**Comment by BlackGhostHD** [comment]
Will be directly implemented on the release branch

**Pull request closed without merging**